jira_sync/config/legacy_model.py

```diff
@@ -0,0 +1,38 @@
+"""Pydantic model describing the legacy configuration file."""
```

Comment:
No need to keep legacy as well as we are the only one using that right now.

Reply:
👍